net-libs/shairplay/shairplay-0_pre20160304.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="6"
+
+inherit autotools 
+
+if [[ ${PV} == "9999" ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="git://github.com/juhovh/shairplay.git"
+else
+	EGIT_COMMIT="498bc5bcdd305e04721f94a04b9f26a7da72673f"
+	SRC_URI="https://github.com/juhovh/shairplay/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz"
+	S="${WORKDIR}/${PN}-${EGIT_COMMIT}"
+fi
+
+DESCRIPTION="Apple airplay and raop protocol server"
+HOMEPAGE="https://github.com/juhovh/shairplay"
+
+LICENSE="BSD LGPL-2.1 MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="static-libs"
+
+RDEPEND="
+	>=dev-libs/openssl-1.0.0
```

Comment:
needs `:0=`, if you have time/commitment, you can also try whether it works with libressl, but don't feel obligated